underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -131,6 +131,12 @@ public void connectFromWorker(Configuration conf, String hostname) {
   public OutputStream create(String path) throws IOException {
     LOG.debug("Create method: {}", path);
     String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    if (newPath.endsWith("_SUCCESS")) {
```

Comment:
I see. That means we need to create an object standing for the directory explicitly, right?

I think a blog or documentation would be very helpful, in particular to the users who are interested in exploring Alluxio over swift on the project website. Could you write some docs for this? It also helps build the integration tests to verify such behavior. Thanks